app/views/components/gds/exportsInputText.scala.html

```diff
@@ -27,9 +27,9 @@
         isPageHeading: Boolean = false,
         headingClasses: String = gdsPageLabel,
         labelClasses: String = "govuk-label--m",
+        defaultInputClasses: String = "govuk-!-width-two-thirds",
```

Comment:
This solution looks OK to me.  What do other devs think?

Reply:
@PatrykRudnicki @PauloPortugal